azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/functions/CleanupResources.java

```diff
@@ -79,12 +78,6 @@ public Boolean apply(String id) {
             boolean deploymentDeleteStatus = false;
 
             if (jobDone) {
-               StorageService ss = api.getStorageAccountApi(group).get(storageAccountName);
-               if (ss != null) {
-                  storageAcctDeleteStatus = api.getStorageAccountApi(group).delete(storageAccountName);
-               } else {
-                  storageAcctDeleteStatus = true;
-               }
```

Comment:
I wonder why are storage accounts no longer deleted? What the provider creates when creating the node, it should also destroy when destroying the node, no?